README.md

```diff
@@ -187,8 +160,94 @@ Logical volume size can be extended, but not reduced -- this is for
+* ensure (Property)
+* alloc (Parameter) - Selects the allocation policy when a command needs to allocate Physical Extents from the Volume Group. Allowed Values:
+    - `:anywhere`
+    - `:contiguous`
+    - `:cling`
+    - `:inherit`
+    - `:normal`
+* extents (Parameter) - The number of logical extents to allocate for the new logical volume. Set to undef to use all available space
+* initial_size (Parameter) - The initial size of the logical volume. This will only apply to newly-created volumes
+* minor (Parameter) - Set the minor number
+* mirror (Property) - The number of mirrors of the volume.
+* mirrorlog (Property) - How to store the mirror log (core, disk, mirrored).  Allowed Values:
+   -  `:core`
+   -  `:disk`
+   -  `:mirrored`
+* no_sync (Parameter) - An optimization in lvcreate, at least on Linux.
+* persistent (Parameter) - Set to true to make the block device persistent
+* readahead (Parameter) - The readahead count to use for the new logical volume.
+* region_size (Parameter) - A mirror is divided into regions of this size (in MB), the mirror log uses this granularity to track which regions are in sync. CAN NOT BE CHANGED on already mirrored volume. Take your mirror size in terabytes and round up that number to the next power of 2, using that number as the -R argument.
+* size (Property) - The size of the logical volume. Set to undef to use all available space
+* size_is_minsize (Parameter) Default value: `:false` - Set to true if the ‘size’ parameter specified, is just the minimum size you need (if the LV found is larger then the size requests this is just logged not causing a FAIL)
+* stripes (Parameter) - The number of stripes to allocate for the new logical volume.
+* stripesize (Parameter) - The stripesize to use for the new logical volume.
+* volume_group (Parameter) - The volume group name associated with this logical volume. This will automatically set this volume group as a dependency, but it must be defined elsewhere using the volume_group resource type.
+
+### physical_volume
+
+* name (Parameter) (Namevar)
+* ensure (Property) -
+* force (Parameter) Default value: `:false` - Force the creation without any confirmation. Allowed Values:
+   - `:true`
+   - `:false`
+* unless_vg (Parameter) - Do not do anything if the VG already exists. The value should be the name of the volume group to check for.
+
+### volume_group
+
+* name (Parameter) (Namevar) - The name of the volume group.
+* ensure (Property)
+* createonly (Parameter) Default value: :false - If set to true the volume group will be created if it does not exist. If the volume group does exist no action will be taken. Defaults to `false`.  Allowed Values:
+   - `:true`
+   - `:false`
+* physical_volumes (Property) - The list of physical volumes to be included in the volume group; this will automatically set these as dependencies, but they must be defined elsewhere using the physical_volume resource type.
+
+
+
+## History
```

Comment:
No need for a history section. Anything that wolud normally go here belongs in a changelog file instead.